repository.json

```diff
@@ -793,6 +793,19 @@
 				}
 			]
 		},
+		{
+			"details":"https://github.com/PythonVoiceCodingPlugin/LSP-pyvoice",
+			"name": "LSP-pyvoice",
+			"author": "mpourmpoulis",
+			"labels":["voice coding","accessibility"],
+			"platforms":["windows","linux","osx"],
```

Comment:
can be omitted if it lists all platforms anyway

Reply:
I initially put it There in order to satisfy their review bot Though it still complains